modules/quickjs/0.1.20250426/presubmit.yml

```diff
@@ -0,0 +1,60 @@
+matrix:
+  platform:
+    - debian10
+    - macos
+    - windows
+  bazel: [ 7.x, 8.x, rolling ]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "@quickjs//:quickjs"
+
+bcr_test_windows_module:
```

Comment:
Only `bcr_test_module` will be picked up, not `bcr_test_windows_module` and `bcr_example_module`, you can move the other two tasks under `bcr_test_module` and specify different [`working_directory`](https://github.com/bazelbuild/continuous-integration?tab=readme-ov-file#using-multiple-workspaces-in-a-single-pipeline).